wdl2cwl/main.py

```diff
@@ -28,18 +28,26 @@
 # eval is not on the official list of reserved words, but it is a built-in function
 
 
-def convert(doc: str) -> cwl.CommandLineTool:
+def convert(doc: str) -> Dict[str, List[str]]:
     """Convert a WDL workflow, reading the file, into a CWL workflow Python object."""
     wdl_path = os.path.relpath(doc)
     doc_tree = WDL.load(wdl_path)
 
     parser = Converter()
 
-    tasks = []
-    for task in doc_tree.tasks:
-        tasks.append(parser.load_wdl_objects(task))
+    dict_of_tasks = {"cwlVersion": "v1.2"}
+    dict_of_tasks["$graph"] = []  # type: ignore
 
-    return tasks[0]
+    for index, task in enumerate(doc_tree.tasks):
+        cwl_task_object = parser.load_wdl_objects(task).save()
+        if len(doc_tree.tasks) == 1:
+            return cwl_task_object
+        if index == 0:
+            # change Id name of the first task
+            cwl_task_object["id"] = "main"
```

Comment:
Why?

Reply:
graph doesn't work without a main id name.

Reply:
With `cwltool` you can execute any item in a `$graph` : `cwltool bcftools.cwl#Sort`

It is true, that if you don't want to include the `#id` part then one of the entries much be called `main`.

Reply:
What can I do instead of changing the first ID to `main`?

Reply:
Delete this line. No reason to change IDs 😀👍

Reply:
But it fails when I try to run --validate

Reply:
`cwltool --validate bcftools.cwl#Sort` and `cwltool --validate bcftools.cwl#Annotate`

Reply:
alright. Cool.